### PR TITLE
Adjust the config file in linix use cases

### PR DESCRIPTION
**Pull request by LinconOssa**
The adjustment is made in the config so that everything does not continue to be done in the interface

### CODE REVIEW

File: src/pages/privileges/outlets/linixUseCase/components/DetailsModal/index.tsx [hunk 1]
```diff
@@ -1,43 +1,35 @@
 import { useState } from "react";
 import { Icon } from "@inube/design-system";
 import { InteractiveModal } from "@components/feedback/InteractiveModal";
-import { UseCase } from "@pages/privileges/outlets/linixUseCase/types";
```
Comment (carin91) [inline src/pages/privileges/outlets/linixUseCase/components/DetailsModal/index.tsx:4]: adjust the imports.

File: src/pages/privileges/outlets/linixUseCase/config/dataUseCases.config.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
+import { Link } from "react-router-dom";
```
Comment (carin91) [inline src/pages/privileges/outlets/linixUseCase/config/dataUseCases.config.tsx:1]: adjust imports.

**Review by carin91 — APPROVED** [review]

File: src/pages/privileges/outlets/linixUseCase/components/DetailsModal/index.tsx (hunk 1 above)
Comment (LinconOssa) [reply src/pages/privileges/outlets/linixUseCase/components/DetailsModal/index.tsx:4]: resolved

File: src/pages/privileges/outlets/linixUseCase/config/dataUseCases.config.tsx (hunk 2 above)
Comment (LinconOssa) [reply src/pages/privileges/outlets/linixUseCase/config/dataUseCases.config.tsx:1]: resolved

**Pull request merged**